src/features/groups/list/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { useTranslation } from 'react-i18next';
+import { useCallback, useMemo, useState } from 'react';
+
+import { useGetTransactionGroupsQuery } from '../api';
+import Header from '../../../components/Header';
+import { Column, PageWrapper } from '../../../components/Containers';
+import { CategoriesWrapper, ContentWrapper } from './index.styled';
+import CategoryTab from './category-tab';
+import GroupRow from './group-row';
+import { ITransactionGroupListItem } from '../types';
+
+interface ICategoryTab {
+  key: string;
+  filter: (items: ITransactionGroupListItem[]) => ITransactionGroupListItem[];
+}
+
+const categoryTabs: ICategoryTab[] = [
+  { key: 'all', filter: (items) => items },
+  { key: 'iLent', filter: (items) => items.filter(i => i.yourPart > 0) },
```

Comment:
should we do that instead of extending our backend filter to pass this param and fetch data from a backend?

Reply:
Default tab is 'All', so we receive all items anyway. We do not need to extend server API

Reply:
We also want to use these filters in the offline mode.

Reply:
Ok, agree